docs/src/main/tut/typeclasses/monad.md

```diff
@@ -112,6 +112,8 @@ statement into the monadic context.
 import cats.implicits._
 
 Monad[List].ifM(List(true, false, true))(List(1, 2), List(3, 4))
+
+Monad[List].ifM(List(true, false, false))(List(1, 2), List(3, 4))
```

Comment:
how about also adding names to parameters?
like 

`Monad[List].ifM(List(true, false, true))(ifTrue = List(1, 2), ifFalse = List(3, 4))`